examples/example_build_planet.py

```diff
@@ -1,4 +1,4 @@
-# BurnMan - a lower mantle toolkit
+    # BurnMan - a lower mantle toolkit
```

Comment:
Accidental indentation? That reminds me, we should update the copyright notice...